backend/src/main/scala/cromwell/backend/validation/package.scala

```diff
@@ -0,0 +1,30 @@
+package cromwell.backend
+
+import lenthall.exception.AggregatedException
+
+import scala.util.{Success, Failure, Try}
+
+package object validation {
+  object TryUtils {
```

Comment:
Yes, from engine. We should put this in Core or Lenthal. I think @Horneth is moving this in his PR.